taxi/models.py

```diff
@@ -0,0 +1,38 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+
+from django.db import models
+from django.db.models import CASCADE
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=63, unique=True)
+
+    def __str__(self):
+        return (
+            f"First name: {self.first_name}, "
+            f"last name: {self.last_name}, "
+            f"license number: {self.license_number}"
+        )
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=63, unique=True)
+    country = models.CharField(max_length=63)
+
+    def __str__(self) -> str:
+        return f"Name: {self.name}, " f"country: {self.country}"
```

Comment:
You are overusing f-string in the __str__ method of the Manufacturer model. You should return the string without using f-string, as it is not necessary here. This is a code style issue.